### DISCUSSION TITLE
Stuck on message "Waiting for Istio ingress IP to become available..."

### DISCUSSION THREAD

**Discussion by dkonyayev** [istio/istio#49434] [Q&A]
We are stuck on this message during Istio installation. I was trying to scan the codebase to find out where this message is coming from, but I cannot find it. Any help finding what is behind this message?

**Comment by ilrudie** [comment]
It seems like maybe you're experimenting in KinD or similar and MetalLB isn't running so the svc isn't being assigned an IP.

**Reply by dkonyayev** [reply]
Is it possible to find this message in the codebase so we can see exactly what is happening?

We are installing on Azure Kubernetes Service.

**Reply by ilrudie** [reply]
I'm not certain where that message is coming from. quick glance didn't turn anything up. Are you using helm or istioctl to install?

Can you check that the svc for the ingress gateway looks correct and is getting an LB IP or post it here(redact the IPs)?

This seems related to or the same as https://github.com/istio/istio/issues/49225 which has Keith engaged already. He's at MS so may be more familiar with AKS and Istio. It may be worth keeping an eye on that.

**Reply by dkonyayev** [reply]
We are using istioctl to install. Here's the svc for ingress gateaway.

![image](https://github.com/istio/istio/assets/15344498/b0e458f6-6337-48d1-8e02-6f761d44b489)

**Reply by ilrudie** [reply]
Thanks for the info. I xref'd this into the issue above to see if a resolution was found there since it seems so similar.

Quite odd that an IP is assigned but it remains stuck waiting.
